google/logging/v2/logging_gapic.yaml

```diff
@@ -161,11 +166,14 @@ interfaces:
     timeout_millis: 30000
     field_name_patterns:
       parent: parent
+    resource_name_treatment: NONE
   - name: GetLogMetric
     flattening:
       groups:
       - parameters:
         - metric_name
+        parameter_resource_name_treatment:
```

Comment:
Why set the method-level setting to NONE and override the only flattened parameter? Is this necessary for the unit tests?